packages/desktop-client/src/components/sidebar.js

```diff
@@ -445,11 +445,8 @@ function Accounts({
 
 function ToggleButton({ style, isFloating, onFloat }) {
   return (
-    <View
-      className="float"
-      style={[style, { flexShrink: 0, color: colors.n5 }]}
-    >
-      <Button type="bare" onClick={onFloat}>
+    <View className="float" style={[style, { flexShrink: 0 }]}>
+      <Button type="bare" onClick={onFloat} style={{ color: colors.n5 }}>
```

Comment:
Sorry one more time. The `color` style prop gets overridden on hover, which I had forgotten. The `color` prop overrides the default color.

```suggestion
      <Button type="bare" onClick={onFloat} color={colors.n5}>
```

Will open a PR shortly to fix a couple other instances of this 😬

Reply:
Thanks!